app/controllers/keywords_controller.rb

```diff
@@ -8,9 +8,10 @@ class KeywordsController < ApplicationController
   before_action :csv?, only: :create
   before_action :csv_keyword_in_range?, only: :create
 
-  def new
+  def index
     render locals: {
-      keyword: Keyword.new
+      keyword: Keyword.new,
+      keywords: Keyword.all.order(keyword: :asc)
```

Comment:
Careful, do we want to load all keyword, think about the user has 1000 keywords :D Should we consider to ad the limitation here, and we could introduce the `pagination` feature on another PR ?

Reply:
And we should load only the current user's keywords, here if we use Keyword.all, that load the another user's keyword, that is not correct @gutakk

Reply:
@andyduong1920 That's totally my mistake. Thanks for coming up with this issue.

Reply:
- Limitation : Fixed in 87c42c575ae020205dde8890b545d556a6540ef4
- Fetch only user's keyword : Fixed in cb8f42aa6c764c83193af48c78db961bb5b42348 3306e7df9e3f8066dfb3dd49cb941a82bd7af7e4